src/main/java/org/roy/loadx/transaction/TransactionPrinter.java

```diff
@@ -1,59 +1,101 @@
 package org.roy.loadx.transaction;
 
 import java.util.ArrayList;
-import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
 
 public class TransactionPrinter {
-	private final TransactionAggregator transactionAggregator;
+	static private class TransactionDataSummaryEntry {
+		private String transactionName;
+		private String average;
+		private String count;
+		
+		public TransactionDataSummaryEntry(String transactionName, String average, String count) {
+			this.transactionName = transactionName;
+			this.average = average;
+			this.count = count;
+		}
 
-	public TransactionPrinter(TransactionAggregator transactionAggregator) {
-		this.transactionAggregator = transactionAggregator;
-	}
+		public String getTransactionName() {
+			return transactionName;
+		}
 
-	private static void println() {
-		System.out.println();
-		System.out.flush();
+		public String getAverage() {
+			return average;
+		}
+
+		public String getCount() {
+			return count;
+		}
 	}
+	
+	static private class TransactionDataSummary {
+		static private final String TRANSACTION_HEADER = "Transaction";
+		static private final String AVERAGE_HEADER = "Average";
+		static private final String COUNT_HEADER = "Count";
+		static private final int MAX_ROW_WIDTH = 78;
```

Comment:
I'm assuming an 80 char terminal, that's what I typically use (but can modified).  The `%n` converter in String.format(), used on line 37, will expand to `\r\n` on Windows and `\n` on *nix so 80 - 2 is needed to prevent unexpected wrapping on max length lines on Windows.